{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specifications/{{cookiecutter.package_name}}.in

```diff
@@ -15,11 +15,15 @@ components:
 
     {{cookiecutter.package_name}}.components:
 
+
+{% raw %}
+{% set location = location.replace(" ", "_").replace("'", "").lower() %}
+{% endraw %}
 configuration:
     input_data:
-        location: United States
+        location: {% raw %}{{ location[0]|upper}}{{location[1:] }}{% endraw %}
```

Comment:
I don't think we ran into this, but we have locations where this would fail, I think

Reply:
Are we enforcing capitalization on the first letter and nothing else? Countries are title-case of sorts, so we need capitals for other words, too. I think we are relying on the input spec to be capitalized correctly for all but the first word. Maybe we should just make that the standard -- capitalization is taken as is (and you need to specify it right)

Reply:
If we do, we should leave a comment at the top of the locations file